server/app.js

```diff
@@ -8,12 +8,11 @@ const bunyanMiddleware = require('bunyan-middleware');
 
 const { logger } = require('./services/logger');
 const createHealthRoute = require('./routes/health');
-const createAssessmentRoute = require('./routes/assessment');
 const createViperRoute = require('./routes/viper');
 const createPrisonerAssessmentsRoute = require('./routes/assessments');
 const index = require('./routes/index');
 
-module.exports = function createApp(db, appInfo, assessmentService, viperService, prisonerAssessmentsService) {
+module.exports = function createApp(db, appInfo, viperService, prisonerAssessmentsService) {
```

Comment:
I know this doesn't matter but it would be nice to have the params to `createApp(...)` to be hash
`{db, appInfo, viperService, prisonerAssessmentsService}` that way the order of the arguments would not mater since there are so many of them.

Reply:
Will do. Good shout.